packages/velo-external-db-core/lib/service/data.spec.js

```diff
@@ -168,6 +171,8 @@ describe('Data Service', () => {
         ctx.itemId = chance.guid()
         ctx.itemIds = Array.from({ length: 10 }, () => chance.guid())
         ctx.total = chance.natural({ min: 2, max: 20 })
+        ctx.field = gen.randomDbField()
+        ctx.filterField = gen.randomFilterBaseOnField(ctx.field.field, ctx.field.type)
```

Comment:
Because of the  query validator, we need to generate a valid filter (a column that exists in schema, and an operation that can be preformed of the column)